cities/berlin.json

```diff
@@ -2288,6 +2320,22 @@
             },
             "type": "Feature"
         },
+        {
+            "geometry": {
+                "coordinates": [
+                    13.308666,
+                    52.4669384
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "<164 Markt (Verkauf von Baby- und Kindersachen)",
```

Comment:
Why is there a special character "<"? Or is that the real name?

Reply:
Looking at the source file in [line 4224](https://github.com/wo-ist-markt/wo-ist-markt.github.io/pull/262/files#diff-077b45c932e5588fa665a45630872a0eR4224) it seems to be a typo. I wrote a message to the data provider to clarify/fix it.

Reply:
I received an answer:
> Es soll tatsächlich das "kleiner-Zeichen" darstellen. Wegen Babysachen. Also kleiner als Größe 164.